src/d3d9/d3d9_swapchain.h

```diff
@@ -197,6 +198,24 @@ namespace dxvk {
 
     std::string GetApiName();
 
+    const Com<D3D9Surface, false>& GetFrontBuffer() const {
+      return m_backBuffers.back();
+    }
+
+    bool HasFrontBuffer() const {
+      if (m_presentParams.SwapEffect != D3DSWAPEFFECT_COPY)
+        return true;
```

Comment:
the statement has `&&`s in prev version, so should be
```cpp
      if (m_presentParams.SwapEffect == D3DSWAPEFFECT_COPY)
        return false;
```
and so on

Reply:
This